1-WebApp-OIDC/1-7-B2C-Embedded/README.md

```diff
@@ -0,0 +1,247 @@
+You can use one of the policies in the [Azure AD B2C Custom Policy Starter Pack](https://github.com/Azure-Samples/active-directory-b2c-custom-policy-starterpack). Follow the [Tutorial: Create user flows in Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-user-flows?pivots=b2c-custom-policy) to deploy such policy.
+
+In your policy, aside from the normal steps in customizing the policy, make sure to add the following in **SignUpOrSignIn.xml**, **PasswordReset.xml**, and **ProfileEdit.xml**, inside the `<RelyingParty>` tag, right after the `<DefaultUserJourney>` tag.
+
+```xml
+<!--
+<RelyingParty>
+  <DefaultUserJourney ReferenceId="SignUpOrSignIn" /> -->
+  <UserJourneyBehaviors> 
+    <JourneyFraming Enabled="true" Sources="https://yourcustomdomain.com https://www.yourcustomdomain.com" /> 
+  </UserJourneyBehaviors>
+<!--
+</RelyingParty> -->
+```
+
+>Note: If you want to allow the custom policies to be used with your development environment, make sure you add **<https://localhost:xxxx>** to the list of sources in the `<JourneyFraming>' tag.
+
+Additionally, make sure the `PublicPolicyUri` element in the `<TrustFrameworkPolicy>` in all policy xml files (including **TrustedFramewordBase.xml** and **TrustedFrameworkExtensions.xml**) is referncing your custom domain, not the instance name. The example below is for **SignUpOrSignin.xml**.
+
+```xml
+PublicPolicyUri="http://login.yourcustomdomain.com/B2C_1A_signup_signin"
+```
+
+### Step 5: Create your own Web app
+
+Now you need to [register your web app in your B2C tenant](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-application), so that it has its own Application ID.
+
+Your web application registration should include the following information:
+
+- Enable the **Web App/Web API** setting for your application.
+- Set the **Reply URL** to `https://www.yourcustomdomain.com/signin-oidc`.
+- Copy the Application ID generated for your application, so you can use it in the next step.
+
+### Step 6: Configure the sample with your app coordinates
+
+1. Open the solution in Visual Studio.
+1. Open the `appsettings.json` file.
+1. Find the assignment for `Instance` and replace the value with your tenant name. For example, `https://login.yourcustomdomain.com`
+1. Find the assignment for `Domain` and replace the value with your Azure AD B2C domain name. For example, `yourtenant.onmicrosoft.com`
+1. Find the assignment for `ClientID` and replace the value with the Application ID from Step 4.
+1. Find the assignment for `SignUpSignInPolicyId` and replace with the name of the `Sign up and sign in` policy you created in Step 4.
+
+```JSon
+{
+  "AzureAdB2C": {
+    "Instance": "https://<your-tenant-name>.b2clogin.com",
```

Comment:
@helshabini code snippet here doesn't seem to match the instructions above (3rd one, instance)

Reply:
Indeed, might be confusing. The instructions say the user has to replace that with their own domain name. Will fix that up.

Reply:
Please also update the `domain` value example in addition to the `instance` value example as noted above.